client/js/store/modules/gameList.js

```diff
@@ -7,6 +7,7 @@ class GameSummary {
     this.name = obj.name ? obj.name : null;
     this.numPlayers = Object.prototype.hasOwnProperty.call(obj, 'players') ? obj.players.length : 0;
     this.status = Object.prototype.hasOwnProperty.call(obj, 'status') ? obj.status : false;
+    this.ranked = Object.prototype.hasOwnProperty.call(obj, 'ranked') ? obj.ranked : false;
```

Comment:
This isn't used yet but adding this to the store will let us display games differently in the list if they are ranked